src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax;
+
+import core.basesyntax.dao.FileReaderCsvImpl;
+import core.basesyntax.dao.FileWriterImpl;
+import core.basesyntax.model.Transaction;
+import core.basesyntax.service.ReportServiceImpl;
+import core.basesyntax.service.TransactionProcessor;
+import core.basesyntax.service.TransactionProcessorImpl;
+import core.basesyntax.service.TransitionConvertorImpl;
+import core.basesyntax.strategy.OperationStrategyImpl;
+import java.util.List;
+
+public class Main {
+    private static final String PIVOT_FILE_NAME = "src/main/resources/pivot.csv";
+    private static final String BALANCE_FILE_NAME = "src/main/resources/database.csv";
+
+    public static void main(String[] args) {
+        TransactionProcessor transactionProcessor =
+                new TransactionProcessorImpl(new OperationStrategyImpl());
```

Comment:
```suggestion
                new TransactionProcessorImpl(new OperationStrategyImpl(handlerMap));
```
you should pass handlers map from main